blog/2023-06-22-openfeature-adoption-story.md

```diff
@@ -0,0 +1,130 @@
+---
+slug: 'openfeature-adoption-story'
+title: 'OpenFeature Adoption Story'
+date: 2023-06-22
+authors: [liran2000]
+description: "OpenFeature is an open standard for feature-flagging. This is an adoption story."
+tags: [client, sdk, adoption]
+draft: false
+---
+
+Our team recently needed to use one of the large feature management cloud services.  
+This service had a documented SDK with usage guidelines.  
+During thinking on how to define the exposed interfaces, while keeping it simple and generic, I started doing some research, and encountered [OpenFeature](https://openfeature.dev).  
+Gladly, the mentioned feature management cloud service created provider implementation for OpenFeature.  
+Let me explain why and how it was adopted.
+
+<!--truncate-->
+
+## Why Feature Management was needed
+
+Our product needed a solution for feature flags:
+* providing a gradual release mechanism and a simple way to define target audiences such as target group or split release
+* Using feature flags for dynamic configuration values which can be changed in runtime dynamically without needing for a new release deployment.
+* Ability for A/B testing - can be done via feature for specific region, specific users etc'.
+
+## Why OpenFeature was chosen to be used
+
+While keeping it simple and generic, I had a few wonders:
```

Comment:
Change line 28 to:
While keeping it simple and generic, we had a few considerations: